src/components/Community/Post.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import Badge from "../common/Badge";
+
+export type TeamScheme =
+  | "kia"
+  | "samsung"
+  | "lg"
+  | "doosan"
+  | "ssg"
+  | "kt"
+  | "nc"
+  | "hanhwa"
+  | "lotte"
+  | "kiwoom";
+
+export interface PostProps {
+  id: number;
+  match_date: string;
+  home_team_name: TeamScheme;
+  away_team_name: TeamScheme;
+  review_short: string;
+  user_id: number;
+  likes: number;
+  comments: number;
+}
```

Comment:
해당 interface 자체는 Post에 대한 타입을 나타내기 때문에 -Props를 접미사로 붙여줄 필요는 없을 것 같아요~ 

이렇게 되면 좋을 것 같네요!
```suggestion
export interface Post {
  id: number;
  match_date: string;
  home_team_name: TeamScheme;
  away_team_name: TeamScheme;
  review_short: string;
  user_id: number;
  likes: number;
  comments: number;
}
```

Reply:
그리고 보통 백엔드에서는 table 컬럼을 snake_case로 많이 작성하다보니 api에서 값이 타입에 정의된 것처럼 넘어올 수 있는데요!
api에서 넘어온 값을 프론트에서 사용하는 형태로 바꾸는 mapper 같은 것을 만들어서 사용해줘도 좋을 것 같습니다!